src/utils/Calculations.js

```diff
@@ -24,18 +25,62 @@ class Calculator extends Converter {
   }
```

Comment:
can you make one function that have switch statement which contain all logic in order to avoid repeating codes